documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,279 @@
+* Adds two specific visualizations to this Contour instance: `line()` and `tooltip()`. The visualizations to add to your Contour chart are all optional, and which ones to include depends on what your final visualization should look like.
+
+Note that unlike many common (static) uses of Contour, this code is *not* rendering the Contour instance just yet.
+
+See more about <a href="http://forio.com/contour/documentation.html#key_concepts" target="_blank">creating a new Contour chart</a>.
+
+**Step 3: Use the Variables Channel to subscribe to the model variable you want to graph.**
+
+In Flow.js, you can use the [Variables Channel](../generated/channels/variables-channel/) to receive notifications when a model variable is updated. Because we want to update our visualization as a model variable changes, we subscribe to the variable. 
+
+	Flow.channel.variables.subscribe('myVariable', function() { });
+
+
+**Step 4: In the callback function for your subscription, update and render the Contour chart.**
+
+The Flow.js [Variables Channel `subscribe()`](../generated/channels/variables-channel/) method's second argument is a callback function, called whenever the model variable is updated. We want to update and re-render our visualization each time the model variable changes:
+
+	Flow.channel.variables.subscribe('myVariable',
+		function(data) {
+			myChartData.push(data.myVariable);
+			myChart.setData(myChartData);
+			myChart.render();
+	});
+
+The argument passed to the callback (`data`) is an object with the name and value of the variable. Here, we only need to add the value to the data set we're graphing: `push(data.myVariable)`.
+
+
+**Extension #1: Graphing multiple variables updated by the model.**
+
+Using the Flow.js [Variables Channel](../generated/channels/variables-channel/), you can subscribe to multiple model variables at once, which allows you to use multiple variables at once in your visualization.
+
+	Flow.channel.variables.subscribe(['cost', 'price'], 
+		function (data) {
+			var composedData = {
+				x: data.cost,
+				y: data.price
+			};
+			mySecondChartData.push(composedData);
+			mySecondChart.setData(mySecondChartData.push);
+			mySecondChart.render();
+		}, { batch: true });
+
+Make sure to pass in the option `{ batch: true }` as the optional third argument to `subscribe()`. When `batch` is `true`, the callback function is only called once (rather than once for each variable to which you are subscribing). The argument passed to the callback (`data`) is an object with the names and values of both variables. The variable `composedData` uses these values to create an (x,y) data point to add to the data set we're graphing.
+
+**Extension #2: Graphing multiple variables entered by the user.**
+
+The approach in Extension #1 works well when `cost` and `price` are both updated automatically as your model advances. You can also use Flow.js and Contour to graph multiple model variables whose values are input by the user.
```

Comment:
Not sure if "extension" is the right term here, I'd reserve that work for cases when you're adding custom attributes/converters or otherwise changing the behavior of Flow itself.  Maybe just "example"?